spec/factories/state_file_nj_intakes.rb

```diff
@@ -133,16 +134,13 @@
       end
       intake.update(overrides)
 
+      intake.synchronize_df_w2s_to_database
```

Comment:
This line seems to be the one causing issues with the PDF spec and the state_return spec. I will probably need some help from CFA debugging why that is

Reply:
@jachan this comment addresses your question about circleci failures